Dolphin scripts/Entrance Randomizer/lib/entrance_rando.py

```diff
@@ -54,27 +54,41 @@ class Choice(IntEnum):
 }
 ```"""
 
+one_way_exits: list[tuple[int, int]] = [
+    (LevelCRC.WHITE_VALLEY, LevelCRC.MOUNTAIN_SLED_RUN),  # the White Valley geyser
+    (LevelCRC.MOUNTAIN_SLED_RUN, LevelCRC.APU_ILLAPU_SHRINE),  # the Apu Illapu Shrine one-way door
+    (LevelCRC.APU_ILLAPU_SHRINE, LevelCRC.WHITE_VALLEY),  # the Apu Illapu Shrine geyser
+    (LevelCRC.CAVERN_LAKE, LevelCRC.JUNGLE_CANYON),  # the Jungle Canyon waterfall
+]
```

Comment:
If you make these NamedTuples, you won't need the `one_way_list = [` list comprehension below.
It'll still fit any `tuple[int, int]` param by covariance. You can also make it a tuple to ensure it's never accidentally mutated.
```suggestion
one_way_exits = (
    Transition(LevelCRC.WHITE_VALLEY, LevelCRC.MOUNTAIN_SLED_RUN),  # the White Valley geyser
    Transition(LevelCRC.MOUNTAIN_SLED_RUN, LevelCRC.APU_ILLAPU_SHRINE),  # the Apu Illapu Shrine one-way door
    Transition(LevelCRC.APU_ILLAPU_SHRINE, LevelCRC.WHITE_VALLEY),  # the Apu Illapu Shrine geyser
    Transition(LevelCRC.CAVERN_LAKE, LevelCRC.JUNGLE_CANYON),  # the Jungle Canyon waterfall
)
```

Reply:
implemented the `suggested change`.